app/screens/settings/about/about.tsx

```diff
@@ -119,6 +121,27 @@ const About = ({componentId, config, license}: AboutProps) => {
     const intl = useIntl();
     const theme = useTheme();
     const styles = getStyleSheet(theme);
+    const serverUrl = useServerUrl();
+    const [loadMetric, setLoadMetric] = useState<number | null>(null);
+
+    useEffect(() => {
+        const fetchLoadMetric = async () => {
+            try {
+                if (license.IsLicensed === 'true') {
+                    const client = NetworkManager.getClient(serverUrl);
+                    const response = await client.getLicenseLoadMetric();
+                    if (response?.load && response.load > 0) {
+                        setLoadMetric(response.load);
+                    }
+                }
+            } catch {
+                // Silently fail if the endpoint is not available
+                // No need to log the error
+            }
+        };
```

Comment:
This should be a remote action and not here directly in the code and should only try and fetch if the connected server has the endpoint, use `isMinimumServerVersion` for that purpose.

Reply:
Thank you, fixed -- let me know if this matches the pattern.